plugins/node.d/http_loadtime

```diff
@@ -102,9 +102,13 @@ cd $TEMPO_DIR || exit 1
 for uri in $urls
 do
     loadtime=`$time_bin -f "%e" wget $wget_opt --header='Accept-Encoding: gzip,deflate' $uri 2>&1`
-
-    esc_uri="$(clean_fieldname "$uri")"
-    echo $esc_uri".value $loadtime"
+    if [ "$?" != "0" ]; then
```

Comment:
This looks reversed. I would expect "time" or "wget" to return 0 on success in a shell (as opposed to perl).

    if [ "$?" = "0" ]; then
      # do good things
    else
      # warn about error
    fi